terraform/cp4aiops/main.tf

```diff
@@ -1,111 +1,88 @@
+# Provider block
 provider "ibm" {
-  version    = "~> 1.12"
-  region     = var.region
   ibmcloud_api_key = var.ibmcloud_api_key
+  region           = var.region
 }
 
-locals {
-  enable_cluster = var.cluster_id == null || var.cluster_id == ""
+# Getting the OpenShift cluster configuration
+data "ibm_resource_group" "group" {
+  name = var.resource_group
 }
 
-module "cluster" {
-  // source = "../../../../ibm-hcbt/terraform-ibm-cloud-pak/roks"
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/roks"
-  enable = local.enable_cluster
-  on_vpc = var.on_vpc
-
-  // General parameters:
-  project_name = var.project_name
-  owner        = var.owner
-  environment  = var.environment
+resource "null_resource" "mkdir_kubeconfig_dir" {
+  triggers = { always_run = timestamp() }
+  provisioner "local-exec" {
+    command = "mkdir -p ${var.cluster_config_path}"
+  }
+}
 
-  // Openshift parameters:
+module "create_cluster" {
+//  source = "../../../terraform-ibm-cloud-pak/modules/roks" # terraform-ibm-cloud-pak/modules/roks
+  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/terraform-0.13/modules/roks"
```

Comment:
This will need to be updated.  The `terraform-0.13` branch has been merged to `main`.

Update to `github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//modules/roks`